Lesson_Materials/Lecture_06_Asynchronous_Execution/index.html

```diff
@@ -274,7 +273,7 @@
 {
   <mark>auto hostAcc = buf.get_host_access(cgh);</mark>
```

Comment:
As I sent you in the exercise corrections email, there is an another issue on this slide:
The definition of hostAcc should be:
               auto hostAcc = bufR.get_host_access();
  or like in the code it is: auto hostAccR = bufR.get_host_access(sycl::read_only);

Reply:
Fixed thanks